code/__HELPERS/radiation.dm

```diff
@@ -35,6 +35,7 @@
 	pulse_information.threshold = threshold
 	pulse_information.chance = chance
 	pulse_information.minimum_exposure_time = minimum_exposure_time
+	pulse_information.turfs_to_process = get_rad_turfs(source, max_range)
```

Comment:
I'd recommend `RANGE_TURFS` here, which uses the internal block() proc, as laid out in the 8 year old profile series done by Rastaf0 <https://gist.github.com/Giacom/be635398926bb463b42a>
It's a bit faster, and also cleaner I think